src/bin-utils/__tests__/parser.js

```diff
@@ -214,20 +214,6 @@ test('if help is called with a script, it shows the help for that script', () =>
   expect(mockGetLogger.mock.info).toHaveBeenCalledTimes(1)
 })
 
-test('if help is called with a script, it shows the help for that script', () => {
```

Comment:
Why was this removed?

Reply:
There was an eslint error(jest/no-identical-title) which drew my attention to it. The test is actually a repetition - https://github.com/kentcdodds/nps/blob/9d695d8feded0eb8654ec09c744257678bbb7cea/src/bin-utils/__tests__/parser.js#L203-L229

Reply:
Oh! Haha 😅 Thanks!